charts/rafiki-auth/Chart.yaml

```diff
@@ -17,10 +17,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.4.0
+version: 0.4.1
```

Comment:
Why only minor?

Reply:
well, I bumped patch version just because it is a small change in the chart. Basically hasn't change a lot, apart from `NODE_ENV` variable that is now filled from values.yaml and default value is old hardcoded value.

I don't know if I want to automate this. Yes, we can check in some action if anything changed in chart files. If it is, we can bump version. But again, which one? :) How to determine from git diff which version do we need to bump?

Me personally would like to have this manually